concourse/pipelines/guest-package-build.jsonnet

```diff
@@ -508,7 +508,7 @@ local buildpackageimagetask = {
                       '-zone=us-central1-a',
                       '-test_projects=compute-image-test-pool-002,compute-image-test-pool-003,compute-image-test-pool-004,compute-image-test-pool-005',
                       '-images=projects/gcp-guest/global/images/debian-10-((.:build-id)),projects/gcp-guest/global/images/debian-11-((.:build-id)),projects/gcp-guest/global/images/debian-12-((.:build-id)),projects/gcp-guest/global/images/centos-7-((.:build-id)),projects/gcp-guest/global/images/rhel-7-((.:build-id)),projects/gcp-guest/global/images/rhel-8-((.:build-id)),projects/gcp-guest/global/images/rhel-9-((.:build-id))',
-                      '-exclude=(image)|(disk)|(security)|(oslogin)|(storage_perf)',
+                      '-exclude=(image)|(disk)|(security)|(oslogin)|(storage_perf)|(gveperf)',
```

Comment:
Let's go with network_perf so that we can represent other network devices in the future (at least virtio.

Reply:
Done.